src/rgw/support/rgw_support_bundle

```diff
@@ -107,10 +106,25 @@ class RGWSupportBundle:
                 outfile = os.path.join(tmp_motr_trace_dir, file)
                 # Convert m0trace file to human readable yaml format
                 SimpleProcess(f'm0trace -i {infile} -Y -o {outfile}.yaml').run()
+                SimpleProcess(f'xz {outfile}.yaml').run() # compress the output file
 
         if stacktrace:
-            # TODO: CORTX-32151 Generate live stack trace for rgw process
-            pass
+            from subprocess import check_output
+            process = '/usr/bin/radosgw'
+            pid = check_output(['pidof', process]).decode('utf-8').split('\n')[0]
+            stacktrace_file = os.path.join(RGWSupportBundle._tmp_src, 'rgw_live_callstack.log')
+            # Collecting running RGW process stack trace using GDB
+            cmd = """gdb --batch --quiet -ex "thread apply all bt full" """\
+                f"""-ex "quit" {process} {pid}"""
+            Log.info(f"generating stack trace for {process} with pid {pid}")
+            output, _, _ = SimpleProcess(cmd).run()
+            with open(stacktrace_file, 'wb') as fd:
+                if output:
+                    fd.write(output)
+                else:
+                    msg = "gdb package missing, Skipping stack trace generation"
```

Comment:
for any error gdb error would be logged. error needs to be just captured and made available on stdout, if its not happening by default with SimpleProcess executing that command, stderr should ideally show that error

Reply:
Ok. Create a defect to correct that in Simple Process implementation